orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -178,9 +178,26 @@ spec:
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/raw'
                 - name: output-prefix
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/processed'
-          # 9) Diff generated data against existing data
-          - name: diff-tables
+          {{- $datasetName := printf "%s_%s" "monster_staging_data" (include "argo.timestamp" .) }}
+          # 9) Create a dataset
+          - name: create-dataset
             dependencies: [process-archive]
+            templateRef:
+              name: create-dataset
+              template: main
+            arguments:
+              parameters:
+                - name: dataset-name
+                  value: {{ $datasetName }}
+                - name: bq-project
+                  value: '{{ .Values.bigquery.stagingData.project }}'
+                - name: dataset-description
+                  value: 'Dataset used by the Monster team for ingest ETL'
+                - name: dataset-expiration
+                  value: '604800000'
```

Comment:
Could we make the dataset prefix, description, and experation all input `Values`? Not sure if we'll ever want to change them, but if we _did_ it would be annoying to have to push a new pipeline version.

Reply:
Thought about this; might as well do it!